modules/persistent-storage-csi-secrets-store-disconnect-environment.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-secrets-store.adoc
+//
+
+:_mod-docs-content-type: CONCEPT
+[id="persistent-storage-csi-secrets-store-disconnect-environment_{context}"]
+= Support for disconnected environments
+
+The {secrets-store-driver} now supports disconnected environment. The {secrets-store-driver} is verified to work in the disconnected clusters with the following supported secrets providers:
+
+* link:https://aws.amazon.com/secrets-manager/[AWS Secrets Manager]
+* link:https://developer.hashicorp.com/vault[HashiCorp Vault]
+* link:https://cloud.google.com/security/products/secret-manager[Google Secrets Manager]
+* link:https://azure.microsoft.com/en-us/products/key-vault/[Azure Key Vault]
+
+To enable communication between {secrets-store-driver} and the secrets manager, configure VPC-endpoints or equivalent connectivity to the corresponding secrets manager service. Additionally, configure VPC-endpoints or connectivity to the OpenID(OIDC) issuer and the Secure Token Service(STS). The exact configuration depends on the secrets provider, the authentication method and the types of disconnected cluster.
```

Comment:
I see mixed usage of `secrets manager` and `secrets provider`.

Also `Additionally, configure VPC-endpoints` sounds redundant to the previous sentence.